modules/javafx.graphics/src/main/java/javafx/scene/text/Text.java

```diff
@@ -1418,10 +1444,27 @@ public boolean isSettable(Text node) {
             public StyleableProperty<TextBoundsType> getStyleableProperty(Text node) {
                 return (StyleableProperty<TextBoundsType>)node.boundsTypeProperty();
             }
-         };
+        };
+
+        private static final CssMetaData<Text,Number> TAB_SIZE =
+                new CssMetaData<Text,Number>("-fx-tab-size",
```

Comment:
I think that type parameters should have space separation after a comma: `Text, Number`, though I've seen without too.

Reply:
Most of the instances where CssMetaData is used in the rest of the file don't have a space. The exceptions seem to be where there is a wildcard in the type parameter.  It could be left as-is simply to maintain consistency within the file.  Though I tend to agree that a space makes sense.

Reply:
Either is fine with me in this case.